analysis/wikitrust_algorithms/text_trust/version.py

```diff
@@ -0,0 +1,251 @@
+"""
+Module containing Version class. Part of the text_trust package.
+
+Eric Vin, 2019-2020
+"""
+
+#Standard library imports
+import math
+from typing import List, Tuple
+
+#text_trust package imports
+from .block import Block
+from .word import Word
+
+#text_diff package imports
+from wikitrust_algorithms.text_diff.edit import Edit
+
+class Version:
+    """
+    Class representing one version of an article at a certain point in time.
+    """
+
+    def __init__(self, word_list: List[Word], edit_list: List[Edit], author_rep: float, \
+                 constants: Tuple[float, float, float]) \
+                 -> None:
+        """
+        Version Constructor:
+            -words: A list of Words present in this Version
+            -edit_list: A list of Edits that map the previous Version
```

Comment:
Why do we need to map the previous version to this one?  I mean, we need it when computing trust or reputations, but why should this be stored with a revision?

Reply:
This was just an implementation choice, though in hindsight it does seem rather arbitrary and while it works well for a single consecutive non parallelized instance, I think it will definitely have it's drawbacks when we deviate from it. Let's discuss on Thursday.